corehq/apps/userreports/views.py

```diff
@@ -1658,6 +1658,55 @@ def subscribe_to_data_source_changes(request, domain, config_id):
     return HttpResponse(status=201)
 
 
+@csrf_exempt
+@require_POST
+@api_auth()
+@require_permission(HqPermissions.view_reports)
+@toggles.SUPERSET_ANALYTICS.required_decorator()
+@api_throttle
+def unsubscribe_from_data_source(request, domain, config_id):
+    repeater = DataSourceRepeater.objects.filter(
```

Comment:
I see that we are querying the repeater again on line [here](https://github.com/dimagi/commcare-hq/pull/34871/commits/535010d8d7b07491054a022f1ebd04f954e24967#diff-317e2709070b8f05f4b8887b06c70baa61461b17e9d461ec584c7a76398a2409R1688) with connection settings as an additional argument.
Is this query needed then ?

Reply:
Good catch! This is not needed